presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.260
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in desugar lambda expression in sql functions that would cause compiler error when sql function references input in lambda expression when session property ``inline_sql_function`` is set to ``false``.
+* Fix multithread writing for fragment result cache  #16455.
+* Add UUID type and operators.
+* Add a new configuration property ``experimental.aggregation-spill-enabled`` to control aggregate spills explicitly. This can be overridden by ``aggregation_spill_enabled`` session property.
+* Add a new configuration property ``experimental.order-by-spill-enabled`` to control order by spills explicitly. This can be overridden by ``order_by_spill_enabled`` session property.
+* Add a new configuration property ``experimental.window-spill-enabled`` to control window spills explicitly. This can be overridden by ``window_spill_enabled`` session property.
+* Add config `fragment-result-cache.max-single-pages-size` to control the max fragement cache file size.
+* Add session property ``query_max_revocable_memory_per_node`` to override existing configuration property ``experimental.max-revocable-memory-per-node``.
+* Add support for Prometheus connector plugin.
+* Introduce a new config ``optimizer.aggregation-if-to-filter-rewrite-enabled`` and its corresponding session property ``aggregation_if_to_filter_rewrite_enabled`` to enable or disable an optimizer rule to improve the query performance of ``IF`` expressions inside aggregation functions.
+* When spill is enabled, enable join spilling by default.  This can be disabled by setting the configuration property ``experimental.join-spill-enabled`` or session property ``join_spill_enabled`` to ``false``.
```

Comment:
How about changing this to -
Enable join spilling by default when spill is enabled. This can be disabled by setting the configuration property ``experimental.join-spill-enabled`` or session property ``join_spill_enabled`` to ``false``.